.vscode/settings.json

```diff
@@ -2,7 +2,6 @@
   "deno.enable": true,
   "deno.lint": true,
   "deno.unstable": true,
-  "deno.importMap": "import_map.json",
```

Comment:
The Deno VS code plugin detects the `deno.jsonc` file automatically, which then detects the import map, so we don't need this anymore.